bpfman/src/bpf.rs

```diff
@@ -635,19 +635,49 @@ impl BpfManager {
                             prog_args.extend(["--pid".to_string(), pid.to_string()])
                         }
 
-                        let status = std::process::Command::new("./target/debug/bpfman-ns")
+                        debug!("calling bpfman-ns to attach uprobe in pid: {:?}", p);
+
+                        // Figure out where the bpfman-ns binary is located
+                        let bpfman_ns_path = if Path::new("./target/debug/bpfman-ns").exists() {
+                            // If we're running natively from the bpfman
+                            // directory, use the binary in the target/debug
+                            // directory
+                            "./target/debug/bpfman-ns"
+                        } else if Path::new("./bpfman-ns").exists() {
+                            // If we're running on kubernetes, the bpfman-ns
```

Comment:
I really don't think we should be bringing k8s specific things into core bpfman... Maybe we make this path configurable?